src/lifecycle.js

```diff
@@ -366,27 +383,67 @@ function triggerLifecycle (target, component) {
  * @returns {undefined}
  */
 function initElements (elements) {
-  var elementsLen = elements.length;
-
-  for (var a = 0; a < elementsLen; a++) {
-    var element = elements[a];
-
+  var a, b, c, d, e;
```

Comment:
Ended up declaring in the loops. That's how I normally do it anyways. I've been declaring vars at the top most of the time and was just seeing how it panned out here. The end goal is to refactor this to be less complex.